### ISSUE TITLE
Design Consideration.

### ISSUE THREAD

**Issue by AndrewVaughn** [rsanchez-wsu/RaiderPlanner#301]
The salutation input on the create new account screen does not allow for punctuation. My initial thought is that this program should follow conventional grammar rules. This should be a team discussion.

**Comment by rsanchez-wsu** [comment]
@AndrewVaughn I agree.  Could you search through the issue history to confirm that we did not already have this discussion?  If so, please mention the issue here so that they are linked.  Whether or not we have previously had this discussion, could you describe some examples of what you consider to be valid and invalid input?
